templates/edit_movies_form.html

```diff
@@ -9,15 +9,15 @@ <h1 class="mb-5">Edit Movie</h1>
 <form method="post">
```

Comment:
I would add action="/movies/{{movie_id}}" after method="post". I tested it in my vs code and it didnt work without it.

Reply:
Thank you, I fixed it